export class NotFoundException extends Error {
  constructor(message = 'O recurso solicitado n√£o foi encontrado.') {
    super(message)
    this.name = 'NotFoundException'
  }
}
